src/js/components/TextInput/index.d.ts

```diff
@@ -42,7 +43,9 @@ export interface TextInputProps {
   value?: string | number;
 }
 
-declare const TextInput: React.ComponentClass<TextInputProps &
-  Omit<JSX.IntrinsicElements['input'], 'onSelect' | 'size' | 'placeholder'>>;
+declare const TextInput: React.ComponentClass<
```

Comment:
I think this should be changed to FC (functional component) vs component class.

Reply:
Absolutely! I can change it.

Reply:
This comment was already addressed on a PR that was merged yesterday, so feel free to resolve the conflicts or roll it back.

Reply:
Done